### DISCUSSION TITLE
how to popup the modal while drag the component?

### DISCUSSION THREAD

**Discussion by senthil177cse** [GrapesJS/grapesjs#4438] [Q&A]
Hi @artf ...
Thanks for the great GrapesJS plugin...
i want to add the modal popup while drag the component like image...

Actually i am creating custom table for that while drag the component i have to set the rows and columns in modal popup. so can you help me for this please?

**Comment by artf** [comment]
Check this discussion https://github.com/artf/grapesjs/discussions/3758#discussioncomment-1300993
